internal/manifests/manifests_test.go

```diff
@@ -391,7 +391,7 @@ spec:
 			It("fails for manifest with invalid yaml format", func() {
 				clusterID := registerCluster().ID
 				fileName := "99-test.yml"
-				invalidYAMLContent := encodeToBase64("not a valid YAML content")
+				invalidYAMLContent := encodeToBase64("not a valid YAML: {{ content }}")
```

Comment:
I don't really understand why this was changed, in the original code the intention was to have the phrase "not a valid YAML content", I don't think `content` is to be treated as a variable in this case.

Note the phrase "not a valid YAML content" doesn't flow well in English grammar and I think it should say "invalid YAML content" instead, but that's a separate issue.

Reply:
```
"not a valid YAML content"
```
parses successfully into a string

> I don't think content is to be treated as a variable in this case.

the issue is with the curly braces, which makes an invalid way of defining a dictionary. Maybe I can update the string to `invalid YAML content: {` to make the "mistake" obvious.